util.py

```diff
@@ -73,14 +73,20 @@ def get_time_origin(filename):
 
     return dt.date(date.year, date.month, date.day)
 
-def col_idx_largest_lat(lats):
+def _gridLatT_from_y_supergrid(y_supergrid):
     """
-    The col index with the largest lat.
+    Extracts an 1D array of latitudes at T-cell centers from the MOM6 supergrid.
+    ref: https://github.com/mom-ocean/MOM6/blob/d68ff1250e3fc8c75bf3bafc60b935fe5f7417a1/src/initialization/MOM_grid_initialize.F90#L168
```

Comment:
A bit more specific:

```suggestion
    ref: https://github.com/mom-ocean/MOM6/blob/d68ff1250e3fc8c75bf3bafc60b935fe5f7417a1/src/initialization/MOM_grid_initialize.F90#L333-L347
```